### ISSUE TITLE
Obtain third party API keys

### ISSUE THREAD

**Issue by 0xdef1cafe** [shapeshift/web#4744]
### Overview

Currently, the swapper doesn't require API keys, and requests to third-party, upstream routes, e.g. 0x or CoWSwap, are made client side.

When we move the swap logic to be server side, we'll immediately hit free tier rate limits.

We'll need API keys, where applicable, for all existing and future swap routes.

### References and additional details

See above.

### Acceptance Criteria

We have obtained API keys, where required, for the following routes

* 0x
* CoWSwap
* THORChain (n/a I believe)
* 1inch
* Osmosis
* LiFi

**Comment by willyogo** [comment]
APIs that don't support Auth:
1. LiFi
2. Osmosis
3. CowSwap (confirming with them but I'm pretty sure this is the case)
4. THORChain

APIs that require auth:
1. 0x (discussing increased rate limits with them now)
2. I think 1inch; their API License says that the public API only supports 30 requests per minute, but doesn't mention an enterprise API. Reaching out to them for clarity

**Comment by 0xdef1cafe** [comment]
![Image](https://github.com/shapeshift/web/assets/88504456/caa40d1c-05a7-4250-913d-3e980ace436d)

this is problematic - @willyogo can you negotiate w/ cowswap, or we're going to have to punt it out of scope?

**Comment by willyogo** [comment]
1inch API key obtained with 10 RPS. asked if they're avail to discuss increased limits for the API

**Comment by gomesalexandre** [comment]
Closing as we got API keys for all APIs - although with low rps - effectively unblocking base functionality. We can always update the API keys if/as we get increased limit keys.
